src/chains/ethereum/ethereum/patches/memdown+6.1.1.patch

```diff
@@ -0,0 +1,53 @@
+diff --git a/node_modules/memdown/memdown.js b/node_modules/memdown/memdown.js
```

Comment:
In the near future ganache is very likely to ship with memdown, and this patch will probably break things for us.

I think a better solution would be monkey-patch (or duck punch, if you'd rather :fist_oncoming: :duck:) a MemDOWN's `_batch` instance. We also need to ensure that the `_batch` instead we patch is actually called by the test in question, otherwise a future change might accidentally stop using the patched version, then later regress into a race condition again, but then it will still pass CI because the race condition doesn't happen very often.


Oh, I also thought of an alternative approach that doesn't require `setTimeout`, if you'd rather try this. We can detect when `_batch` is called form without our tests and use this to ensure everything happens in the right order. I haven't thought through it fully, so maybe this is worse than simulating a slow file system.

Reply:
I've changed the approach to monkey-patch the memdown instance rather than using patch-package. It seems to work consistently, so let's go with this for now!